cfme/tests/cloud_infra_common/test_cloud_init_provisioning.py

```diff
@@ -95,6 +96,8 @@ def test_provision_cloud_init(appliance, request, setup_provider, provider, prov
         floating_ip = mgmt_system.get_first_floating_ip(pool=ip_pool)
         provider.refresh_provider_relationships()
         inst_args['environment'] = {'public_ip_address': floating_ip}
+        inst_arg_props = inst_args.setdefault('properties', {})
+        inst_arg_props['instance_type'] = partial_match(provisioning['ci-flavor-name'])
```

Comment:
Why is `partial_match` necessary here? Just curious.

Reply:
Yes it is. Other provisioning code uses this partial_match. I tried without and it didn't work. I didn't study the problem well, but I think on the prov page, the option includes the flavor details (the VM memory, cpus and stuff), not just the name of the flavor.